### PR TITLE
[manageiq] added managiq/postgresql.conf.d/*.conf to manageiq log collection

### PR DESCRIPTION
**Pull request by rh-dluong**
---
Please place an 'X' inside each '[]' to confirm you adhere to our [Contributor Guidelines](https://github.com/sosreport/sos/wiki/Contribution-Guidelines)

- [x] Is the commit message split over multiple lines and hard-wrapped at 72 characters?
- [X] Is the subject and message clear and concise?
- [X] Does the subject start with **[plugin_name]** if submitting a plugin patch or a **[section_name]** if part of the core sosreport code?
- [X] Does the commit contain a **Signed-off-by: First Lastname <email@example.com>**?

### CODE REVIEW

File: sos/plugins/manageiq.py [hunk 1]
```diff
@@ -73,6 +73,8 @@ def setup(self):
         ])
         self.add_copy_spec("/var/log/tower.log")
 
+        self.add_copy_spec("/etc/manageiq/postgresql.conf.d/*.conf")
```
Comment (TurboTurtle) [inline sos/plugins/manageiq.py]: Is there nothing else in `/etc/manageiq/` that's worth collecting?

Comment (rh-dluong) [reply sos/plugins/manageiq.py]: Nope, it looks like it was only added to hold additional postgres confs as of the latest major release.  Thanks for the inquiry though!

**Comment by bmr-cymru** [comment]
Please also follow the [contributor guidelines](https://github.com/sosreport/sos/wiki/Contribution-Guidelines) for patch formatting: the subject needs to be in `[<component>] change description` format, hard wrapped to 72 characters and with a brief and reasonable description of the change in the subject line.

E.g.:

```
[manageiq] add postresql configuration path

The most recent version of CloudForms adds postgres configuration
files at:

  /etc/managiq/postgresql.conf.d/

Include these in the set of files collected by the plugin.

Signed-off by: David Luong <dluong@redhat.com>
```

You can fix your commit with `git commit --amend`, and then re-push the branch with `git push --force`.

**Comment by rh-dluong** [comment]
Sorry about that @bmr-cymru , I thought that subject was referring to the PR itself and not the commit message, will keep that in mind for the future, thanks!

File: sos/plugins/manageiq.py (hunk 1 above)
Comment (bmr-cymru) [reply sos/plugins/manageiq.py]: In future, if you're adding a new item following an existing `add_copy_spec()` call, please just add it as a list - there's no need to make duplicate calls:

```python
                self.add_copy_spec([
                    "/var/log/tower.log",
                    "/etc/manageiq/postgresql.conf.d/*.conf"
                ])
```

It's more readable and makes the code easier to modify in future. I will fix this up as I merge in this case.

Comment (rh-dluong) [reply sos/plugins/manageiq.py]: Thanks for the pointers, I'll try and keep that in mind next time.

**Pull request closed without merging**